fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalQuickSort.java

```diff
@@ -30,56 +30,30 @@
 import org.apache.commons.lang3.StringUtils;
 
 import java.util.List;
-import java.util.Objects;
 import java.util.Optional;
 
 /**
- * Physical sort plan.
+ * Physical quick sort plan.
  */
-public class PhysicalHeapSort<CHILD_TYPE extends Plan> extends PhysicalUnary<CHILD_TYPE> {
+public class PhysicalQuickSort<CHILD_TYPE extends Plan> extends AbstractPhysicalSort<CHILD_TYPE> {
```

Comment:
Does "QuickSort" here stand for quick sort algorithm? It seems that this node is irrelevant to sort algorithm chosen by BE.

Reply:
without limit and offset, be use quick sort to do sort indeed